From db66082c3463de8f139f9224454195b8c72a5bbd Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Tue, 7 Mar 2023 12:05:49 +0100
Subject: [PATCH 1104/1141] arm64: defconfig: add CoreSight support in addons
 fragment

Enable the driver and the feature required to use the CoreSight
components in fragment-02-defconfig-addons.config.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Ic47d1012518e87a0657218a848e76a5d2bb09646
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/353645
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 0be62da50df5..00d70ac85b75 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -9,6 +9,13 @@ CONFIG_BT_LE=y
 CONFIG_CAN_M_CAN=m
 CONFIG_CAN_M_CAN_PLATFORM=m
 CONFIG_CLKSRC_STM32_LP=y
+CONFIG_CORESIGHT=y
+CONFIG_CORESIGHT_CPU_DEBUG=y
+CONFIG_CORESIGHT_CTI=y
+CONFIG_CORESIGHT_LINK_AND_SINK_TMC=y
+CONFIG_CORESIGHT_SINK_TPIU=y
+CONFIG_CORESIGHT_SOURCE_ETM4X=y
+CONFIG_CORESIGHT_STM=y
 CONFIG_COUNTER=m
 CONFIG_CRYPTO_DEV_STM32_CRC=m
 CONFIG_CRYPTO_DEV_STM32_CRYP=m
-- 
2.39.2

